plugins/config_manager.py

```diff
@@ -0,0 +1,241 @@
+#
```

Comment:
Well, this error is explicitly disabled in our PyLint script. That could mean that we don't require them...